wp-content/themes/access/src/js/modules/screener.js

```diff
@@ -152,9 +152,39 @@ class Screener {
       this._goToStep(this._$steps[0]);
     }
 
+    this._scenarioAnalysis();
+
     return this;
   }
 
+  /**
+   * View Tracking for Webtrends and Google Analytics
+   */
+  _scenarioAnalysis() {
+    let key = '';
+    let data = [];
+
+    $(window).on('hashchange', function() {
+      let hash = window.location.hash;
+      let step = $(hash);
+      key = step.data('trackKey');
+      data = step.data('trackData');
```

Comment:
The next two lines seem to be fired regardless, so we can remove them from the if statement